docs/conformance.md

```diff
@@ -1,14 +1,43 @@
 # Conformance testing
 
+The Conformance tests are subset of our e2e test suite that defines set of features
+defining Kubevirt core functionality. The goal of the suite is to make sure that selected features work across different Vendors, CNIs, CSIs, CRIs implementations.
+
+## How are Conformance tests selected
+
+By default tests with a `[Conformance]` tag (and from release 0.59 also decorator) will be executed and tests with a
+`[Disruptive]` tag will be skipped. These tests will not destroy or modify
+anything outside of the explicitly created test namespaces by the binary.
+
+The tests are always part of Kubevirt e2e tests that are usually approved by individual [Sigs](https://github.com/kubevirt/community/blob/main/sig-list.md#special-interest-groups). For the Conformance tests we have higher requirments than the regular tests, most significant requirments are:
+- Test is not Disruptive
+- Test is not flaky and is reliable per data available from CI team
+- For more concreate details see below
+
+
+## Conformance tests requirments:
+- The feature being tested is GA (no feature gate)
+- The feature is not deprecated
+- The feature is turned on by default (no feature toogle)
+- The test uses only GAed Kubernetes APIs/features
+- The test uses only APIs available to user
```

Comment:
Should the eventual goal here to also only use GA'd KubeVirt features without FGs? I appreciate that isn't possible right now so the current wording is fine but would be nice to document something here.

Reply:
See first point "- The feature being tested is GA (no feature gate)"

Reply:
ACK apologies missed that.

Reply:
Thank you for the review. I can add the reasoning if some points need to.